com.unity.formats.fbx/Editor/ExportModelSettings.cs

```diff
@@ -95,6 +95,15 @@ public override void OnInspectorGUI ()
             exportSettings.SetExportUnredererd(EditorGUILayout.Toggle(exportSettings.ExportUnrendered));
             EditorGUI.EndDisabledGroup ();
             GUILayout.EndHorizontal ();
+            
+            GUILayout.BeginHorizontal();
+            EditorGUILayout.LabelField(new GUIContent("Preserve Import Settings for Existing FBX",
+                "If checked, the import settings from the overwritten FBX will be carried over to the new version."), GUILayout.Width(LabelWidth - FieldOffset));
```

Comment:
Is this the "hover over" text? As you can tell, I don't know much about the Unity GUI API :-)

Reply:
Yes, it's a description that pops up when you hover over it.